### DISCUSSION TITLE
Default App Launch

### DISCUSSION THREAD

**Discussion by D4RKNOISE** [Budibase/budibase#2605] [Ideas]
It would be useful if you could select a default app. It opens immediately when a user logs in.

**Comment by joebudi** [comment]
This is an interesting thought and I can see the benefit of it. Thanks!

**Comment by KernOil** [comment]
I like this idea. I would love separate URLs or to be able to set a default app for each user if they are only allowed to access one app. this is where the no access profile would come in. if the user only has one app they log in and will directly be put into the app.

**Comment by nishantnaagnihotri** [comment]
@joebudi This would really help me with my app configuration.
I am in a blocker situation, because it is infeasible for somebody to give app access to each and every user.

Is there any plan to roll out a solution soon?

**Comment by shogunpurple** [comment]
https://docs.budibase.com/docs/link-directly-to-budibase-app
